brainiak/fcma/classifier.py

```diff
@@ -502,3 +528,35 @@ def decision_function(self, X=None):
             self.test_data_ = self._prepare_test_data(normalized_corr_data)
         confidence = self.clf.decision_function(self.test_data_)
         return confidence
+
+    def score(self, X, y, sample_weight=None):
+        """ returns the mean accuracy on the given test data and labels
+
+        NOTE: This function will not work in sklearn.svm.SVC with precomputed
+        kernel when the kernel matrix is computed portion by portion.
+
+        Parameters
+        ----------
+        X: a list of numpy array in shape [num_TRs, self.num_voxels\_]
```

Comment:
You refer to an actual variable, `self.num_voxels_`. However, you use `num_TRs` as a placeholder next to it, simply to indicate that TRs should be the first axis. I find this a bit confusing. How about choosing names that are clearly not variable names, e.g., `[TRs, voxels]`, and explaining that voxels must be `self.num_voxels_`?

Clearly this is a larger discussion, affecting all our documentation:
https://mail.scipy.org/pipermail/scipy-dev/2013-January/018261.html
There is no conclusion.

I brought it up here because I noticed the difference in roles between the two shape element names.